participants.csv

```diff
@@ -1,3 +1,3 @@
 Zaccharie Ramzi,
 Guillaume Daval-Frérot,
-
+Mathilde Ripart,
```

Comment:
Similarly to https://github.com/zaccharieramzi/git-tuto/pull/2#discussion_r352586153, I would rather keep this new line at the end of the file.